# HG changeset patch
# User Dan Villiom Podlaski Christiansen <danchr@gmail.com>
# Date 1747225899 -7200
#      Wed May 14 14:31:39 2025 +0200
# Branch stable
# Node ID 809a4c4725436a6f3755e58cb864faed4264ad25
# Parent  8c0474a25ff0c788b147ce6381ffbb2435c1c4c2
# EXP-Topic patches
rust_darwin

diff --git a/rust/.cargo/config.toml b/rust/.cargo/config.toml
--- a/rust/.cargo/config.toml
+++ b/rust/.cargo/config.toml
@@ -5,3 +5,15 @@
 # (which would be loaded first).
 [target.'cfg(target_os = "windows")']
 rustflags = ["-Ctarget-feature=+crt-static"]
+
+[target.x86_64-apple-darwin]
+rustflags = [
+  "-C", "link-arg=-undefined",
+  "-C", "link-arg=dynamic_lookup",
+]
+
+[target.aarch64-apple-darwin]
+rustflags = [
+  "-C", "link-arg=-undefined",
+  "-C", "link-arg=dynamic_lookup",
+]
